test/import-map.test.js

```diff
@@ -0,0 +1,106 @@
+const request = require("supertest");
+const { app, setConfig } = require("../src/web-server");
+
+beforeAll(() => {
+  setConfig({
+    manifestFormat: "importmap",
+    username: "username",
+    password: "password",
```

Comment:
Why have a username and password for tests? I only did so for the health endpoint because I was specifically testing authentication related behavior. However, since these tests don't test auth, they don't need it.

```suggestion
```